### ISSUE TITLE
Auto-tag variants submitted to MME

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#2938]
**Is your feature request related to a problem? Please describe.**
Occasionally, variants are submitted to MME and not tagged or tagged and not submitted. 

**Describe the solution you'd like**
For better tracking, we would like to remove the ability to manually add a MME tag and instead automatically add a MME tag once a variant has been submitted. If ever removed from MME, the tag will also need to go away. 

**Describe alternatives you've considered**
NA

**Additional context**
Discussed with Chrissy.

**Comment by hanars** [comment]
What should we do with all the variants currently tagged with the MME tag that are not in matchmaker?

**Comment by lynnpais** [comment]
I like the idea of the new MME Submission tag!

To efficiently use the space/ reduce tags with little value, I am thinking of removing the older seqr MME tag. We will lose data on candidates that have a seqr MME tag but were never submitted. The analysts don't think this is of much value. Before you get rid of the old tag, would it be possible to give us a list of cases where the gene was never submitted but has a seqr MME tag?

**Comment by hanars** [comment]
How about we remove the old tag if the variant is actually submitted to matchmaker but leave it if the variant is not yet in matchmaker, and don't allow users to add it to any new variants. Then using the summary data page the analysts can review all the variants with the tag that aren't in matchmaker at their leisure and remove it where relevant

**Comment by lynnpais** [comment]
I think that would be confusing esp to collaborators - why some genes have a seqr MME tag and some do not. Also if the gene is no longer a candidate, there's no way for us to remove the tag, adding to confusion.

**Comment by hanars** [comment]
The old tag would be a normal tag - you remove it the way you remove any other tag, you just couldn't add it back later. I am not so comfortable just bulk removing a bunch of tags, especially given that in the past they.have been used for some other tracking purposes. What if we rename it "seqr MME (old)" for the first pass of this feature and then after this functionality is live we fully remove it after a couple weeks, once we've had time to review everything that is left with that tag

**Comment by lynnpais** [comment]
Yes, this sounds good. To review, (i) all genes submitted get a new MME
submission tag, (ii) the old seqr MME tag is renamed to seqr MME (old), and
(iii) the old seqr MME tag can no longer be added but can be removed if
needed.

**Comment by anneodonnell** [comment]
if there is an easy way get a list of all genes/samples with an MME tag that is not actually submit to MME, that would be handy to share with the analysts so that they can clean these up. Reasonable to do by looking in each project but there are a lot of project so if this is a relatively easy list to generate, I think they would appreciate it.
